index.js

```diff
@@ -17,6 +17,15 @@ function fullscreen(el) {
   var vendors = ['', 'webkit', 'moz', 'ms', 'o']
 
   for(var i = 0, len = vendors.length; i < len; ++i) {
+    // if internet explorer
+    if (i === 3) {
+        changeEventName = 'MSFullscreenChange';
```

Comment:
Few issues: 

- `changeEventName` and `errorEventName` don't seem to be defined anywhere beforehand.
- `changeEventName` and `errorEventName` don't end up getting used in `ael.call`